arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -0,0 +1,757 @@
+          ]
+        },
+        "upgradeMode": {
+          "type": "string",
+          "enum": [
+            "Default",
+            "Automatic",
+            "Manual"
+          ]
+        },
+        "clientCertificateThumbprints": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientCertificateThumbprint"
+          }
+        },
+        "clientCertificateCommonNames": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientCertificateCommonName"
+          }
+        },
+        "fabricSettings": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SettingsSectionDescription"
+          }
+        },
+        "reverseProxyCertificate": {
+          "$ref": "#/definitions/CertificateDescription"
+        },
+        "managementEndpoint": {
+          "type": "string",
+          "readOnly": true
+        },
+        "nodeTypes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NodeTypeDescription"
+          }
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Default",
+            "WaitingForNodes",
+            "Deploying",
+            "BaselineUpgrade",
+            "UpdatingUserConfiguration",
+            "UpdatingUserCertificate",
+            "UpdatingInfrastructure",
+            "EnforcingClusterVersion",
+            "UpgradeServiceUnreachable",
+            "Deleting",
+            "ScaleUp",
+            "ScaleDown",
+            "AutoScale",
+            "Ready",
+            "Failed"
+          ]
+        },
+        "vmImage": {
+          "type": "string",
+          "readOnly": true
+        },
+        "diagnosticsStorageAccountConfig": {
+          "$ref": "#/definitions/diagnosticsStorageAccountConfig"
+        },
+        "upgradeDescription": {
+          "$ref": "#/definitions/paasClusterUpgradePolicy"
+        }
+      }
+    },
+    "clusterPatchParameters": {
+      "description": "Cluster patch parameters",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/clusterResourceProperties"
+        },
+        "propertyNames": {
+          "description": "Allowed properties",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "ReliabilityLevel",
+              "ClusterUpgradeMode",
+              "ClientCertificateThumbprints",
+              "FabricSettings",
+              "ReverseProxyCertificate",
+              "NodeTypes"
+            ]
+          }
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "x-ms-azure-resource": false
+    },
+    "ClusterPatchRequest": {
+      "description": "Cluster patch request",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/clusterPatchParameters"
+        }
+      }
+    },
```

Comment:
Did not quite fellow, last time i tried to do this, and ARM failed my request, not sure what i have missed when generated .net SDK library.